src/extension.ts

```diff
@@ -179,17 +180,33 @@ export function activate(context: vscode.ExtensionContext) {
         return;
       }
 
-      const uri = vscode.Uri.parse(`couchbase:/${node.bucketName}/${node.scopeName}/${node.collectionName}/${documentName}.json`);
-      memFs.writeFile(
-        uri,
-        Buffer.from("{}"),
-        { create: true, overwrite: true }
-      );
+        const uri = vscode.Uri.parse(`couchbase:/${node.bucketName}/${node.scopeName}/${node.collectionName}/${documentName}.json`);
```

Comment:
Looks like your indentation has got mixed up. This should be one increment to the left. Same for the closing brackets on 209.